Source/WebKit/UIProcess/Extensions/WebExtension.cpp

```diff
@@ -2095,7 +2096,7 @@ RefPtr<JSON::Object> WebExtension::bestIconVariantJSONObject(RefPtr<JSON::Array>
             continue;
 
         RefPtr variantObject = variant->asObject();
-        auto colorSchemes = toColorSchemes(variantObject ? variantObject->getValue(colorSchemesManifestKey) : nullptr);
+        auto colorSchemes = toColorSchemes(variantObject ? variantObject->getValue(colorSchemesManifestKey) ?: variantObject->getValue(colorSchemesAPIKey) : nullptr);
```

Comment:
```suggestion
        auto colorSchemes = toColorSchemes(variantObject ? variantObject->getValue(colorSchemesAPIKey) ?: variantObject->getValue(colorSchemesManifestKey) : nullptr);
```

This would have to be swapped around to support passing both casings at the same time, while still favouring the non deprecated `colorSchemes`.

Reply:
the variant object will only have either `colorSchemes` or the `color_schemes` key.  and `colorSchemes` takes precedence when we parse the dictionary so the order doesn't matter here.